Makefile

```diff
@@ -226,4 +212,4 @@ $(DEPS_BUILD_DIR)/%: phony_force
 	$(BUILD_TOOL) -C $(DEPS_BUILD_DIR) $(patsubst $(DEPS_BUILD_DIR)/%,%,$@)
 endif
 
-.PHONY: test lualint pylint shlint functionaltest unittest lint clint clean distclean nvim libnvim cmake deps install appimage checkprefix commitlint
+.PHONY: test lualint pylint shlint functionaltest unittest lint clint clean distclean nvim libnvim cmake deps install appimage checkprefix commitlint oldtest helptags helphtml benchmark clint-full functionaltest-lua generated-sources all check-single-includes
```

Comment:
Might be useful to split this and declare a new `.PHONY` after a bunch of targets.